monitoring/mongodb/alerts.yaml

```
@@ -139,3 +139,45 @@ groups:
     annotations:
       description: 'MongoDb has low disk space'
       summary: 'MongoDb has low disk space'
+  
+  - alert: MongoDbNodeRecovering
```

Node in a k8s/artesca context means a logical server, where we can have multiple MongoDB instances running (e.g. the mongod, mongos)

```suggestion
  - alert: MongoDBInstanceRecovering
```

(maybe we want to use "Pod" instead of "Instance" as it also has a meaning in mongodb)